src/client/pydaos/pydaos_core.py

```diff
@@ -180,13 +175,13 @@ def get_kv_by_name(self, name, root=None, create=False):
         return new_kv
 
     def __str__(self):
-        return '{}@{}'.format(self.cuuid, self.puuid)
+        return '{}@{}'.format(self.cont, self.pool)
```

Comment:
Question: Curious why we use `cont@pool` here, but, for example, C duns_resolve_path uses `pool/cont`.
Should these be synchronized at some point?

Reply:
Agreed. I am actually fixing this in a follow-up patch that i will upload once this one lands.